contracts/modules/ApprovedTransfer.sol

```diff
@@ -113,4 +149,13 @@ contract ApprovedTransfer is BaseTransfer {
         uint numberOfSignatures = 1 + Utils.ceil(guardianStorage.guardianCount(_wallet), 2);
         return (numberOfSignatures, OwnerSignature.Required);
     }
+
+    /**
+    * @dev Helper to Reset the daily consumption.
+    * @param _wallet The target wallet.
+    */
+    function doResetLimit(address _wallet) internal {
```

Comment:
This is probably better placed in the `LimitUtils` ?

Reply:
Done